Makefile

```diff
@@ -402,11 +402,17 @@ else
 endif
 .PHONY: fmt/shfmt
 
-lint: lint/shellcheck lint/go
+lint: lint/shellcheck lint/go lint/ts lint/helm
 .PHONY: lint
 
+lint/ts:
+	cd site
+	yarn && yarn lint
+.PHONY: lint/ts
+
 lint/go:
 	./scripts/check_enterprise_imports.sh
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.53.2
```

Comment:
Can this be skipped if golangci-lint is in PATH?

Reply:
Why do it though? It runs almost instantly if already installed.